docs/layouts/shortcodes/regions.html

```diff
@@ -24,6 +25,7 @@ <h4><a href="#{{ $regionAnchor }}" id="{{ $regionAnchor }}" aria-label="Anchor">
             </td>
             <!-- Provider -->
             <td>{{ .provider }}</td>
+            <td>{{ if .available }}✅{{ end }}</td>
```

Comment:
:bulb:
Do we want to add a ⚠️ for the ones that are not directly available?
Since in some cases, it's still possible for customers, but it can be seen as a reminder that not all features are available + a further incentive to consider migration.

Reply:
Also, regions that aren't available might still have all the features. There's a difference between `legacy` and not `available`